setup.py

```diff
@@ -245,6 +245,8 @@ def get_outputs(self):
       for filename in [
           f for f in filenames if fnmatch.fnmatch(
               f, "*.so") or fnmatch.fnmatch(f, "*.py")]:
+        if filename.endswith("_proto.so"):
+          continue
```

Comment:
Could you note how this is related to the grpc package? Also, is it possible to match a pattern with "grpc" in the filename, just in case some other package has a dependency on a library also ending in "proto"?

Reply:
I see, that explains it very well, thanks!